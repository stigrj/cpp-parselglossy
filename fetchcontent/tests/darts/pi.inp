title = "This program computes Pi"
method = darts

Darts {
  n_darts = 100000
  random_seed = 225512
}

