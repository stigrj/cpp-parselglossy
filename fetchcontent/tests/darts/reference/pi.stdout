
-------------- JSON input -----------------
{
  "Darts": {
    "n_darts": 100000,
    "random_seed": 225512
  },
  "Integration": {
    "n_steps": 1000
  },
  "method": "darts",
  "title": "This program computes Pi"
}
-------------------------------------------

-------------- JSON output ----------------
{
  "method": "Throwing darts",
  "pi": 3.14152,
  "title": "This program computes Pi"
}
-------------------------------------------

